Cake.VsCode.Recipe/Content/build.cake

```diff
@@ -120,7 +120,7 @@ BuildParameters.Tasks.NpmInstallTask = Task("Npm-Install")
 {
     var settings = new NpmInstallSettings();
     settings.LogLevel = NpmLogLevel.Silent;
-    NpmInstall(settings);
+    NpmCI(settings);
```

Comment:
@nils-a should we go one step further and do an if/else here?  i.e. check to see if we are running a local build and if we are doing a full npm install, and do npm ci only when running on CI?